app/parsers/bulkrax/csv_parser.rb

```diff
@@ -211,16 +211,13 @@ def export_headers
       # we don't want access_control_id exported and we want file at the end
       # also sort the headers so they're grouped and easier to find
       headers.delete('access_control_id') if headers.include?('access_control_id')
-      headers.delete('file') if headers.include?('file')
```

Comment:
no longer adding `file` and `collections` here because they are now numerated and in `self.headers`. I imported a csv I exported last week (in a zip along with the files) and it imported correctly too.

Reply:
that makes sense to me.

Reply:
this makes sense to me